core/Pos/Core/Update/Types.hs

```diff
@@ -296,6 +300,20 @@ mkSystemTag tag | T.length tag > systemTagMaxLength
                 | otherwise
                     = pure $ SystemTag tag
 
+osHelper :: OS -> String
+osHelper sys = case sys of
+    Windows -> "win"
+    OSX     -> "macos"
+    Linux   -> "linux"
+    _       -> display sys
+
+archHelper :: Arch -> String
+archHelper archt = case archt of
+    I386    -> "32"
+    X86_64  -> "64"
+    _       -> display archt
```

Comment:
I think it's better to put these helpers into `Pos.Core.Update.Util`, not `Pos.Core.Update.Types`. `Types` is primarily for type definitions (unsurprisingly).